src/tests/java/picard/analysis/CollectGcBiasMetricsTest.java

```diff
@@ -38,136 +35,87 @@
     private final static String library1 = "TestLibrary1";
     private final static String library2 = "TestLibrary2";
     private final static String library3 = "TestLibrary3";
+    private final static int LENGTH = 99;
+    private final static int NUM_READS = 100;
+    private final static String READ_NAME = "TESTBARCODE";
 
     private final static File TEST_DIR = new File("testdata/picard/sam/CollectGcBiasMetrics/");
-    private final File dict = new File(TEST_DIR, "Mheader.dict");
+    private final File dict = new File(TEST_DIR, "MNOheader.dict");
 
-    File tempSamFile;
+    File tempSamFileChrM_O;
+    File tempSamFileAllChr;
 
     SAMRecordSetBuilder setBuilder1 = new SAMRecordSetBuilder(true, SAMFileHeader.SortOrder.coordinate);
     SAMRecordSetBuilder setBuilder2 = new SAMRecordSetBuilder(true, SAMFileHeader.SortOrder.coordinate);
     SAMRecordSetBuilder setBuilder3 = new SAMRecordSetBuilder(true, SAMFileHeader.SortOrder.coordinate);
+    SAMRecordSetBuilder setBuilder4 = new SAMRecordSetBuilder(true, SAMFileHeader.SortOrder.coordinate);
```

Comment:
the setBuilder.addPair takes in a contig number, and a start and end location for each read pair to the reference and the start and end location is determined from a 'random' integer but it always uses the same ones because of `final Random rg = new Random(5);` in setup below.